Snowflake.Data/Core/SFReusableChunk.cs

```diff
@@ -74,10 +81,10 @@ private class BlockResultData
             private static readonly int NULL_VALUE = -100;
             private int blockCount;
 
-            private static int blockLengthBits = 24;
+            private static int blockLengthBits = 23;
             private static int blockLength = 1 << blockLengthBits;
             int metaBlockCount;
-            private static int metaBlockLengthBits = 15;
+            private static int metaBlockLengthBits = 17;

```

Comment:
why do we change this?

Reply:
I did some tests and we have hundreds of meta blocks. By increasing the size of a block we reduce the number of blocks. But in JDBC there is 15, so I can revert it

Reply:
Ok, let's revert it.